libtelemetry/src/main/java/com/mapbox/android/telemetry/VisionEventFactory.java

```diff
@@ -30,11 +32,32 @@ public Event createVisionEvent(Event.Type type) {
     return BUILD_EVENT_VISION.get(type).build();
   }
 
+  public VisionAttachment createVisionAttachment(String filePath, MediaType mediaType,
+                                                  AttachmentMetadata attachmentMetadata) {
+    return buildVisionAttachment(filePath, mediaType, attachmentMetadata);
+  }
+
+  public Attachment createAttachment(Event.Type type) {
+    checkVisionEvent(type);
+    return buildAttachment();
+  }
+
   private VisionEvent buildVisionEvent() {
     VisionEvent visionEvent = new VisionEvent();
     return visionEvent;
   }
 
+  private VisionAttachment buildVisionAttachment(String filepath, MediaType mediaType,
```

Comment:
What is the benefit of maintaining a set of private wrapper methods that simply return the instance of an object?